pkgs/top-level/all-packages.nix

```diff
@@ -7996,8 +7996,11 @@ in
   python37Full = python37.override{x11Support=true;};
 
   # pythonPackages further below, but assigned here because they need to be in sync
+  _merge_pythonPackages = true;
```

Comment:
camelCase please.

Reply:
:thinking: this isn't a package, so it shouldn't follow package naming guidelines. This is an attribute lifted out of it's attribute set to prevent evaluations. Maybe `pythonPackages_merge_ = true;` would look better?

Reply:
We are more often use camelCase for function then for packages.